xgi/classes/reportviews.py

```diff
@@ -375,7 +375,8 @@ def duplicates(self):
         Notes
         -----
         The IDs returned are in an arbitrary order, that is duplicates are not
-        guaranteed to be consecutive.
+        guaranteed to be consecutive. For IDs with the same bipartite neighbors,
+        only the first ID added is not a duplicate.
```

Comment:
```suggestion
        guaranteed to be consecutive.
        If two IDs have the same bipartite neighbors, only one of them is returned.
```

Reply:
I wonder if we should add an option to return all of them, not only those ones that are actually repeated (and ignoring the first one). In a way, `duplicates` is similar to a `groupby` operation (it's grouping IDs that have the same bipartite neighbors) and `groupby` operations always return all of the elements involved, they never ignore the first one.

Reply:
Or am I overthinking things again? :sweat_smile:

Reply:
Added as issue [#163](https://github.com/ComplexGroupInteractions/xgi/issues/163).